apis/tempo/v1alpha1/microservices_types.go

```diff
@@ -94,6 +91,17 @@ type MicroservicesList struct {
 	Items           []Microservices `json:"items"`
 }
 
+// Resources defines resources configuration.
+type Resources struct {
```

Comment:
I am introducing this wrapper to enable us in the future add configuration of the split for the resources.

 e.g.

```yaml
cpu:
  ingester: 0.5m
  distributor: 0.2m
```

Reply:
Ive not strong opinion yet. Would it be sufficient to support only a `%` option?

Reply:
> Would it be sufficient to support only a % option?

@frzifus sorry I don't understand what do you mean?